src/main/java/seedu/weme/logic/parser/util/ParserUtil.java

```diff
@@ -71,6 +71,7 @@ public static WemeParser forContext(ModelContext modelContext) {
         case CONTEXT_CREATE:
             return CREATE_PARSER;
         case CONTEXT_STATISTICS:
+        case CONTEXT_VIEW:
         case CONTEXT_PREFERENCES:
             // TODO: This is a temporary placeholder until all tabs have been implemented
             return new WemeParser() {
```

Comment:
problem here, the view context should only support exit, help, and unview or whatever it will be called. Need to move undo redo to the individual parsers since we are doing it like this.